### ISSUE TITLE
A question regarding emotiv_values and emotiv_data values.

### ISSUE THREAD

**Issue by AkumaShogun** [openyou/emokit#286]
I want to use the data from the emotiv but I need its values in Voltage.

I've read the documentation thoroughly and I've found myself with a question. What's the difference between the data written by having write_decrypted and write_values set as True?

If write_values is set as true it writes the sensor and quality values and saves them to a 'emotiv_values_*.csv' file, of which I've obtained the following data: 

Timestamp | F3 Value | F3 Quality | FC5 Value | FC5 Quality | F7 Value | F7 Quality | T7 Value | T7 Quality | P7 Value | P7 Quality | O1 Value | O1 Quality | O2 Value | O2 Quality | P8 Value | P8 Quality | T8 Value | T8 Quality | F8 Value | F8 Quality | AF4 Value | AF4 Quality | FC6 Value | FC6 Quality | F4 Value | F4 Quality | AF3 Value | AF3 Quality | X Value | Y Value | Z Value
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
2018-11-23 13:10:55.985753 | 4305.12121169 | 0 | 4179.42424201 | 0 | 4308.21212078 | 0 | 4221.15151473 | 0 | 4452.96969652 | 0 | 4369.51515108 | 0 | 4277.81818139 | 0 | 4496.24242379 | 0 | 3766.78787841 | 0 | 4423.60606016 | 0 | 4214.45454503 | 0 | 4289.15151472 | 0 | 4155.2121208 | 0 | 4124.81818141 | 0 | 42 | 42 | ?
2018-11-23 13:10:55.987023 | 4305.12121169 | 0 | 4179.42424201 | 0 | 4308.21212078 | 0 | 4221.15151473 | 0 | 4452.96969652 | 0 | 4369.51515108 | 0 | 4277.81818139 | 0 | 4496.24242379 | 0 | 3766.78787841 | 0 | 4423.60606016 | 0 | 4214.45454503 | 0 | 4289.15151472 | 0 | 4155.2121208 | 0 | 4124.81818141 | 0 | 42 | 42 | ?
2018-11-23 13:10:55.988165 | 4305.12121169 | 0 | 4179.42424201 | 0 | 4308.21212078 | 0 | 4221.15151473 | 0 | 4452.96969652 | 0 | 4369.51515108 | 0 | 4277.81818139 | 0 | 4496.24242379 | 0 | 3766.78787841 | 0 | 4423.60606016 | 0 | 4214.45454503 | 0 | 4289.15151472 | 0 | 4155.2121208 | 0 | 4124.81818141 | 0 | 42 | 42 | ?
2018-11-23 13:10:55.989936 | 4350.96969653 | 0 | 4149.03030262 | 0 | 4322.6363632 | 0 | 4197.45454503 | 0 | 4443.69696925 | 0 | 4361.78787835 | 0 | 4310.27272684 | 0 | 4507.57575713 | 0 | 3777.60606023 | 0 | 4426.18181774 | 0 | 4202.60606019 | 0 | 4304.09090866 | 0 | 4174.78787837 | 0 | 4068.15151474 | 0 | 42 | 42 | ?
2018-11-23 13:10:55.993383 | 4322.12121169 | 0 | 4122.24242383 | 0 | 4735.78787831 | 0 | 4171.18181776 | 0 | 4449.36363592 | 0 | 4337.06060563 | 0 | 4337.57575714 | 0 | 4466.87878743 | 0 | 3779.66666629 | 0 | 4398.36363592 | 0 | 4102.15151474 | 0 | 4292.75757533 | 0 | 4170.15151473 | 0 | 4033.12121172 | 0 | 42 | 42 | ?
2018-11-23 13:10:55.994890 | 4322.12121169 | 0 | 4122.24242383 | 0 | 4735.78787831 | 0 | 4171.18181776 | 0 | 4449.36363592 | 0 | 4337.06060563 | 0 | 4337.57575714 | 0 | 4466.87878743 | 0 | 3779.66666629 | 0 | 4398.36363592 | 0 | 4102.15151474 | 0 | 4292.75757533 | 0 | 4170.15151473 | 0 | 4033.12121172 | 0 | 42 | 42 | ?
2018-11-23 13:10:55.997045 | 4280.39393897 | 0 | 4213.93939352 | 0 | 4349.9393935 | 0 | 4215.99999958 | 0 | 4475.12121167 | 0 | 4350.45454502 | 0 | 4318.51515108 | 0 | 4482.33333289 | 0 | 3769.8787875 | 0 | 4396.30302986 | 0 | 4299.45454502 | 0 | 4282.45454503 | 0 | 4180.45454504 | 0 | 4141.30302989 | 0 | 42 | 42 | ?
2018-11-23 13:10:55.998922 | 4270.09090866 | 0 | 4194.87878746 | 0 | 4382.39393896 | 0 | 4172.72727231 | 0 | 4520.96969652 | 0 | 4347.87878744 | 0 | 4297.90909048 | 0 | 4492.12121167 | 0 | 3748.75757538 | 0 | 4409.69696926 | 0 | 4351.99999956 | 0 | 4270.60606018 | 0 | 4146.45454504 | 0 | 4210.33333291 | 0 | 42 | 42 | ?
2018-11-23 13:10:56.000971 | 4270.09090866 | 0 | 4194.87878746 | 0 | 4382.39393896 | 0 | 4080.51515111 | 0 | 4520.96969652 | 0 | 4330.87878745 | 0 | 4297.90909048 | 0 | 4492.12121167 | 0 | 3748.75757538 | 0 | 4409.69696926 | 0 | 4351.99999956 | 0 | 4270.60606018 | 0 | 4146.45454504 | 0 | 4210.33333291 | 0 | 42 | 42 | ?
2018-11-23 13:10:56.004599 | 4238.66666624 | 0 | 4187.66666625 | 0 | 4459.66666622 | 0 | 4134.60606019 | 0 | 4518.39393894 | 0 | 4345.30302987 | 0 | 4291.7272723 | 0 | 4482.33333289 | 0 | 3745.15151478 | 0 | 4410.72727229 | 0 | 4362.30302987 | 0 | 4255.15151473 | 0 | 4120.69696928 | 0 | 4230.424242 | 0 | 42 | 42 | ?
2018-11-23 13:10:56.006204 | 4258.24242382 | 0 | 4171.18181776 | 0 | 4500.87878743 | 0 | 4234.54545412 | 0 | 4530.7575753 | 0 | 4374.66666623 | 0 | 4315.42424199 | 0 | 4496.75757531 | 0 | 3737.42424205 | 0 | 4407.6363632 | 0 | 4178.90909049 | 0 | 4262.36363594 | 0 | 4129.45454504 | 0 | 4186.63636322 | 0 | 42 | 42 | ?
2018-11-23 13:10:56.012223 | 4278.84848442 | 0 | 4206.21212079 | 4.1212121208 | 4586.90909045 | 0 | 4262.87878745 | 0 | 4498.81818137 | 0 | 4377.2424238 | 0 | 4365.90909047 | 0 | 4501.90909046 | 0 | 3749.78787841 | 0 | 4375.18181774 | 0 | 4063.51515111 | 0 | 4272.66666624 | 0 | 4145.93939352 | 0 | 4146.45454504 | 0 | 42 | 42 | ?
2018-11-23 13:10:56.014117 | 4278.84848442 | 0 | 4206.21212079 | 4.1212121208 | 4586.90909045 | 0 | 4262.87878745 | 0 | 4498.81818137 | 0 | 4377.2424238 | 0 | 4365.90909047 | 0 | 4501.90909046 | 0 | 3749.78787841 | 0 | 4375.18181774 | 0 | 4063.51515111 | 0 | 4272.66666624 | 0 | 4145.93939352 | 0 | 4146.45454504 | 0 | 42 | 42 | ?
2018-11-23 13:10:56.023237 | 4278.84848442 | 0 | 4206.21212079 | 4.1212121208 | 4586.90909045 | 0 | 4262.87878745 | 0 | 4498.81818137 | 0 | 4377.2424238 | 0 | 4365.90909047 | 0 | 4501.90909046 | 0 | 3749.78787841 | 0 | 4375.18181774 | 0 | 4063.51515111 | 0 | 4272.66666624 | 0 | 4145.93939352 | 0 | 4146.45454504 | 0 | 42 | 42 | ?
2018-11-23 13:10:56.030015 | 4260.81818139 | 0 | 4240.21212079 | 4.1212121208 | 4501.90909046 | 0 | 4271.12121169 | 0 | 4512.21212076 | 0 | 4384.45454502 | 0 | 4425.66666622 | 0 | 4502.93939349 | 0 | 3761.12121175 | 0 | 4342.72727229 | 0 | 4047.03030263 | 0 | 4288.12121169 | 0 | 4170.66666625 | 0 | 4141.8181814 | 12.3636363624 | 42 | 42 | ?

However, if write_decrypted is set as True, it says that it writes the data received after decrypting it. And then saves this data to a 'emokit_data_*.csv' file.
How is this decrypted data different from the sensor values and quality values? Given that I obtain something similar to this when write_decrypted is enabled:

2018-11-22 15:34:03.093292 | 89 | 129 | 38 | 0 | 247 | 228 | 161 | 111 | 129 | 2 | 37 | 8 | 79 | 0 | 0 | 0 | 2 | 9 | 24 | 131 | 220 | 106 | 134 | 98 | 2 | 232 | 16 | 223 | 83 | 107 | 104 | 187
-- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | -- | --
2018-11-22 15:34:03.093769 | 109 | 129 | 26 | 0 | 151 | 228 | 33 | 111 | 128 | 242 | 35 | 136 | 72 | 192 | 0 | 0 | 2 | 9 | 40 | 130 | 220 | 103 | 134 | 78 | 1 | 200 | 19 | 95 | 73 | 109 | 104 | 191
2018-11-22 15:34:03.094144 | 110 | 129 | 18 | 0 | 135 | 228 | 161 | 111 | 128 | 250 | 35 | 152 | 72 | 192 | 0 | 0 | 2 | 8 | 120 | 130 | 220 | 99 | 134 | 46 | 1 | 168 | 19 | 31 | 58 | 109 | 105 | 183
2018-11-22 15:34:03.096510 | 111 | 129 | 18 | 0 | 183 | 230 | 97 | 102 | 129 | 6 | 35 | 72 | 72 | 128 | 0 | 0 | 2 | 8 | 184 | 128 | 156 | 102 | 134 | 58 | 1 | 184 | 19 | 31 | 55 | 109 | 106 | 179
2018-11-22 15:34:03.098126 | 112 | 128 | 250 | 0 | 167 | 231 | 33 | 103 | 129 | 2 | 35 | 88 | 72 | 64 | 0 | 0 | 2 | 9 | 136 | 129 | 156 | 105 | 134 | 86 | 2 | 8 | 19 | 159 | 70 | 109 | 106 | 251

So, to recap, I'd like to know the difference between these two options.

As of now, I've assumed that the 'emotiv_values_*.csv' file was the correct one since I can somewhat make sense of it if I substract 4096 from it and multiply it by 0.51 according to the resolution and LSB value of the emotiv. Am I correct in this assumption?
